tests/fixtures/common/collections/ansible_collections/testorg/coll_1/plugins/lookup/lookup_1.py

```diff
@@ -1,9 +1,8 @@
-"""
-The get_path lookup plugin
-"""
+"""An ansible test lookup plugin."""
+
 from __future__ import absolute_import, division, print_function
 
-__metaclass__ = type
+__metaclass__ = type  # pylint: disable=invalid-name
```

Comment:
By the way, it's possible to just add this to valid names, I think.

Reply:
OTOH, do we even need to support Python 2 here?

Reply:
no, but ansible-test will still complain if it's missing

```
ERROR: Found 1 metaclass-boilerplate issue(s) which need to be resolved:
ERROR: plugins/modules/mod_1.py:0:0: missing: __metaclass__ = type
```

and I forgot, you can't even disable the pylint error in the same line, ergh.

Reply:
For ansible-test this can be done via `sanity.txt`.

Reply:
agreed it could, but generally we try to minimize that file